README.md

```diff
@@ -1,73 +1,39 @@
-mobile
-====
+# mobile
+
+## Overview
 
-Overview
-----
 * This project uses the [OpenDota API](https://docs.opendota.com/) to show data on your mobile device (Android or iOS).
 
 <img src="https://i.imgur.com/3YlIvId.png" height="320" width="180"/> <img src="https://i.imgur.com/64FC8jR.png" height="320" width="180"/> <img src="https://i.imgur.com/JXYHXMx.png" height="320" width="180"/>
 
 <img src="https://i.imgur.com/A3Zbo6Q.png" height="320" width="180"/> <img src="https://i.imgur.com/U9bveMh.png" height="320" width="180"/>
 
+## Download
 
+The App is available now on iOS and Android:
 
-Beta is Open
-----
-How to get invited:
-1. Head over to our [discord](https://discord.gg/0o5SQGbXuWCNDcaF).
-2. Post a message on the mobile channel or send a private message to Proyoyo with your:
-    - Email address
-    - Device type (e.g. iPhone 8, Samsung Galaxy S8+, etc)
-3. You will receive an email from Mobile Center Team (Android) or TestFlight (iOS) with the invitation and download instruction.
-
-Please submit bug reports to Github issue page. If there's any question, feel free to post in our discord mobile channel/send a private message to Proyoyo in discord. 
-
-Mobile Center Download Instruction
-----
-1. Open the invitation email from Mobile Center Team and click "See Details".
-<img src="https://i.imgur.com/WiUFiFA.png" height="640" width="360"/>
-
-2. Create a new account to Mobile Center or login using existing credentials. **WARNING! Do not login using Google. There's a bug with AppCenter that hasn't been fixed by Microsoft.**
-<img src="https://i.imgur.com/xmFQtFf.png" height="640" width="360"/>
-
-3. Click Download. **WARNING! Do not click on "Latest version is available here" as it does not always get the latest available version.**
-<img src="https://i.imgur.com/ejopAVs.png" height="640" width="360"/>
-
-4. Install the downloaded apk.
-
-TestFlight Download Instruction
-----
-1. Open the invitation email from TestFlight and click "View in TestFlight".
-<img src="https://i.imgur.com/AEUduMb.png" height="239" width="318"/>
+<a href="https://itunes.apple.com/us/app/opendota/id1354762555?mt=8"><img alt='Download on the App Store' src='https://linkmaker.itunes.apple.com/assets/shared/badges/en-us/appstore-lrg.svg' width="135"/></a>
+<a href='https://play.google.com/store/apps/details?id=com.opendota.mobile&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1' ><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png' width="135"/></a>
```

Comment:
Can we add the Google Play badge to a different line? It looks janky because it has padding while Apple's doesn't.